experimental/packages/opentelemetry-exporter-metrics-otlp-http/src/platform/index.ts

```diff
@@ -14,7 +14,4 @@
  * limitations under the License.
  */
 
-/* eslint no-restricted-syntax: ["warn", "ExportAllDeclaration"] --
- * TODO: Replace export * with named exports before next major version
- */
-export * from './node';
+export { OTLPMetricExporter } from './node';
```

Comment:
Yep that's somewhat confusing but it's how all our platform-specific exports work, see [here](https://github.com/open-telemetry/opentelemetry-js/blob/b95a4a8fc1243c63c8cadc902824387203f39999/experimental/packages/opentelemetry-exporter-metrics-otlp-http/package.json#L10-L15). 🙂  

The way I understand it, when bundled for the the browser, webpack overrides all usages of this file with the one from `platform/browser/index.js` - so the user will end up with the correct implementation. 

AFAIK we can't override the other way around so that's what ended up becoming the default.